doc-content/enterprise-only/cases/case-management-creating-manager-decision-milestone-proc.adoc

```diff
@@ -4,13 +4,11 @@ This milestone is reached when the `managerDecision` variable has been given a r
 
 .Procedure
 . In the process designer, expand *Milestone* in the *Object Library* and drag a new milestone on the canvas below the *Notify requestor* script task.
-. Click the new milestone and in the upper-right corner, click image:cases/left-arrows.png[] to open the *Properties* panel.
+. Click the new milestone and in the upper-right corner, click the *Properties* image:getting-started/diagram_properties.png[] icon.
 . Input `Manager decision` in the *Name* field.
-. Select `true` from the *Adhoc autostart* menu.
-. Click the down arrow in the *Assignments* field and add the following new conditions in the *Manager decision Data I/O* window.
-+
-Add `CaseData(data.get("managerDecision") != null)` by selecting *Constant* from the *Source* menu.
-
+. Expand *Implementation/Execution* and select *Adhoc Autostart*.
+. Expand *Data Assignments*, click image:getting-started/diagram_properties.png[] in the *Assignments* field, and add the following new condition in the *Manager decision Data I/O* window.
+. Click the *Source* column drop-down, select *Constant*, and input `org.kie.api.runtime.process.CaseData(data.get("managerDecision") != null)`.
```

Comment:
I would just say 'menu' but you can say drop-down menu.

Reply:
I just have to be me